tests/keras_tests/feature_networks_tests/feature_networks/manual_bit_selection.py

```diff
@@ -165,8 +165,8 @@ def get_tpc(self):
         quantization_configurations.extend([
             tpc.layer2qco[tf.multiply].base_config.clone_and_edit(activation_n_bits=4),
             tpc.layer2qco[tf.multiply].base_config.clone_and_edit(activation_n_bits=2)])
-        tpc.layer2qco[tf.multiply] = replace(tpc.layer2qco[tf.multiply], base_config=base_config,
-                                             quantization_configurations=tuple(quantization_configurations))
+        tpc.layer2qco[tf.multiply] = tpc.layer2qco[tf.multiply].model_copy(
```

Comment:
what's `model_copy`? is it a pydantic method? is it replacing the clone_and_edit ins some way?

Reply:
it is a method to copy the instance with the new attributes (since the original instance is immutable)